gsitemap.php

```diff
@@ -546,6 +549,73 @@ protected function getCategoryLink(&$link_sitemap, $lang, &$index, &$i, $id_cate
         return true;
     }
 
+    /**
+	 * return the link elements for the manufacturer object
+	 *
+	 * @param array  $link_sitemap    contain all the links for the Google Sitemap file to be generated
+	 * @param array  $lang            language of link to add
+	 * @param int    $index           index of the current Google Sitemap file
+	 * @param int    $i               count of elements added to sitemap main array
+	 * @param int    $id_manufacturer manufacturer object identifier
+	 *
+	 * @return bool
+	 */
+	protected function getManufacturerLink(&$link_sitemap, $lang, &$index, &$i, $id_manufacturer = 0)
+	{
+		$link = new Link();
+		if (method_exists('ShopUrl', 'resetMainDomainCache')) {
+			ShopUrl::resetMainDomainCache();
+        }
+        
+        // Get manufacturers IDs
+		$manufacturers_id = Db::getInstance()->ExecuteS('SELECT m.`id_manufacturer` FROM `' . _DB_PREFIX_ . 'manufacturer` m
+			INNER JOIN `' . _DB_PREFIX_ . 'manufacturer_lang` ml on m.`id_manufacturer` = ml.`id_manufacturer`' .
+			($this->tableColumnExists(_DB_PREFIX_ . 'manufacturer_shop') ? ' INNER JOIN `' . _DB_PREFIX_ . 'manufacturer_shop` ms ON m.`id_manufacturer` = ms.`id_manufacturer` ' : '').
```

Comment:
We can remove all calls to tableColumnExists along with that method. The Db structure is the same since 1.7.1.0.

There is even a typo and it checks for ($this->tableColumnExists(_DB_PREFIX_ . 'supplier_shop') in other place in the code. :-)))

Reply:
Ok, I'll remove them.

True, will change the supplier_shop for cms_shop too for this PR (will add in the description)